pkg/menu/menu.go

```diff
@@ -22,7 +22,12 @@ type Entry interface {
 	// If there is many default Entrys, choose the the first in the list.
 	IsDefault() bool
 	// Exec performs the following process after an entry is chosen
-	Exec() error
+	Exec(<-chan ui.Event) error
```

Comment:
I know that you did this so that it was possible to test, but it feels strange to have this parameter... @rminnich any thoughts on how to test without having to do this? I'll keep thinking about it too

Reply:
why  not put it in the structs that use it, instead of making it a parameter?